src/components/layouts/collection-page-layout.tsx

```diff
@@ -558,46 +558,37 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
               {!isEmpty(podcast) && (
                 <CoursePodcast podcast={podcast} instructorName={name} />
               )}
-              <div
-                className={`grid grid-cols-1 md:gap-x-5 ${
-                  coursePrerequisites && courseTopics
-                    ? 'md:grid-cols-2'
-                    : 'md:grid-cols-1'
-                }`}
-              >
```

Comment:
Removes two columns on prereq & course topics to optimize for legibility and consistency. 

![CleanShot 2021-06-15 at 07 11 00@2x](https://user-images.githubusercontent.com/57044804/122067963-da8ff780-cda8-11eb-86f5-05dfb6b50b83.png)